extension/js/content_scripts/webmail/gmail/gmail-element-replacer.ts

```diff
@@ -643,9 +643,12 @@ export class GmailElementReplacer extends WebmailElementReplacer {
     const newReplyBoxes = $('div.nr.tMHS5d, td.amr > div.nr, div.gA td.I5').not('.reply_message_evaluated').filter(':visible').get();
     if (newReplyBoxes.length) {
       this.replyOption = undefined;
+      // Try to get message id from plain reply box
+      // https://github.com/FlowCrypt/flowcrypt-browser/issues/5906
+      const replyBoxMessageId = $(newReplyBoxes[0]).find('[name="lts"]').val() as string;
```

Comment:
`[name="lts"]` seems to have id of the first thread message, as I've tested composing encrypted reply to the third thread message and quoted text included only first message content.

but there is another hidden input `[name="rm"]`, and looks like it have id of correct reply message. tried to use it in the code and reply message contained correct quoted text.

Reply:
Hmm, interesting - after I've replaced `[name="lts"]` with `[name="rm"]` switching to encrypted reply works correctly in some threads, while in others quote text is just `undefined`, probably because of error you've mentioned.